src/js/components/Notification/Notification.js

```diff
@@ -0,0 +1,71 @@
+import React, { useContext } from 'react';
+import { ThemeContext } from 'styled-components';
+import { defaultProps } from '../../default-props';
+
+import { Layer } from '../Layer';
+import { Box } from '../Box';
+import { Button } from '../Button';
+import { Text } from '../Text';
+import { Paragraph } from '../Paragraph';
+
+const Notification = ({ toast, message, body, status, onClose }) => {
+  const theme = useContext(ThemeContext) || defaultProps.theme;
+
+  const { icon: CloseIcon } = theme.notification.button;
+  const { icon: StatusIcon, color } = theme.notification[status];
+  const iconColor = theme.notification.icon.color;
+
+  let content = (
+    <Box direction="row">
+      <Box
+        {...theme.notification.iconContainer}
+        background={color}
+        justify="center"
+      >
+        <StatusIcon color={iconColor} />
+      </Box>
+      <Box
+        {...theme.notification.textContainer}
+        align="start"
+        direction="row"
+        justify="between"
+        fill
+      >
+        <Box>
+          <Text {...theme.notification.messageText}>{message}</Text>
+          {body && (
+            <Paragraph {...theme.notification.bodyText}>{body}</Paragraph>
+          )}
+        </Box>
+        {onClose && (
+          <Button
+            {...theme.notification.button}
+            icon={<CloseIcon color={iconColor} />}
+            onClick={onClose}
+            plain
+          />
+        )}
+      </Box>
+    </Box>
+  );
+
+  if (toast) {
+    content = (
+      <Layer animation="fadeIn" modal={false} onEsc={onClose}>
```

Comment:
Should theme or props be able to control the position of the layer? for example, 
```
<Notification position="top-right" />
```

Reply:
yeah this is the next step i think. @ericsoderberghp and i talked about allowing users to pass in layer props via the toast prop as an object to control layer-specific things like position and animation

Reply:
potentially allow the caller to specify a ref as well (e.g. I might want that position to be specified in reference to a particular element in my page layout such a below a header nav, top of main, etc.)